app/src/main/java/tech/pauly/findapet/discover/AnimalDetailsActivity.kt

```diff
@@ -22,5 +23,9 @@ class AnimalDetailsActivity : BaseActivity() {
 
         setSupportActionBar(findViewById(R.id.toolbar))
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
+
+        val typeface = ResourcesCompat.getFont(this, R.font.quicksand_bold)
```

Comment:
This sucks. Setting color and size and stuff works fine through XML, but not font?? So this is a hack.